src/rsa.rs

```diff
@@ -432,7 +432,12 @@ enum BIGNUM {}
 
 #[cfg(feature = "rsa_signing")]
 #[allow(non_camel_case_types)]
-enum BN_BLINDING {}
+#[repr(C)]
+struct BN_BLINDING {
+    a: *mut BIGNUM,
+    ai: *mut BIGNUM,
+    counter: u32,
+}
```

Comment:
I would like to avoid this, if we can, especially since it is only for the sake of testing. Responding below re alternatives.